Makefile

```diff
@@ -0,0 +1,45 @@
+start: ## Start the docker containers
+	@echo "Starting the docker containers"
+	@docker-compose up -d
+	@echo "Containers started - http://localhost:8000"
+
+stop: ## Stop Containers
+	@docker-compose down
+
+build: ## Build Containers
+	@docker-compose build
+
+ssh: ## SSH into running web container
+	docker-compose exec web bash
+
+migrations: ## Create DB migrations in the container
+	@docker-compose exec web python manage.py makemigrations
+
+migrate: ## Run DB migrations in the container
+	@docker-compose exec web python manage.py migrate
+
+shell: ## Get a Django shell
+	@docker-compose exec web python manage.py shell
+
+superuser: ## Get a Django shell
+	@docker-compose exec web python manage.py createsuperuser
+
+test: ## Run Django tests
+	@docker-compose exec web python manage.py test
+
+init: start migrate superuser  ## Quickly get up and running (start containers, migrate DB, create superuser)
```

Comment:
`init` doesn't run `npm install`. Should it?

Reply:
Oops, yes it should. Good catch! https://github.com/dimagi/mpact/pull/24